test/unit-tests/$.test.js

```diff
@@ -117,5 +117,18 @@ describe(test_name, () => {
       const elems = await $('#foo').elements();
       expect(await elems[0].text()).to.be.eql('taiko');
     });
+
+    it('test text() with element index', async () => {
```

Comment:
Can we have a test where there are more than one element?
An id is unique so ideally we should be match by something else.